src/webgpu/util/f32_interval.ts

```diff
@@ -141,6 +142,11 @@ function toF32Vector(v: number[] | F32Vector): F32Vector {
   unreachable(`Cannot convert [${v}] to F32Vector`);
 }
 
+/** @returns the input + zero if any of the entries are subnormal, otherwise returns the input */
```

Comment:
I'd avoid `+` here, as it looks like an addition instead of a concatenation.

Reply:
Done